packages/showcases/vue/stories/components/overlays/VtmnToast/VtmnToast.stories.js

```diff
@@ -22,6 +24,14 @@ export default {
         type: 'boolean',
       },
     },
+    timeout: {
+      type: { name: 'number', required: false },
+      description: 'Duration of the animation',
+      defaultValue: 4500,
+      control: {
+        type: 'number',
+      },
+    },
```

Comment:
Same here.
You don't need to add this, and use the one from the CSF file instead.